cli/.isort.cfg

```diff
@@ -0,0 +1,13 @@
+[settings]
```

Comment:
do we need to duplicate this file under the cli dir or the copy in the root dir is enough when running with tox as before?

Reply:
I didn't realize we had one in the root directory to be honest.

When I try to move the changes into that file the config doesn't seem to have an effect. Placing it in the CLI dir seems to work.